### PR TITLE
Quantum circuit converters

### LINKED ISSUE #2: Adding circuit plotting

### PR DESCRIPTION
**Pull request by madagra**
Convert a quantum circuit built with PyQ into Qiskit. This is useful for visualization for example.

Closes #2

### CODE REVIEW

**Comment by Brogis1** [comment]
Nice 👍 Thanks a lot @madagra !

File: pyqtorch/converters/store_ops.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    def clear(self, *args, **kwargs):
+        """Clear the current circuit visualization"""
+        self.operations = []
+        self.nqubits = 0
+
+
+ops_cache = OpsCache()
+
+
+def storable(func):
```
Comment (awennersteen) [inline pyqtorch/converters/store_ops.py:35]: Running this on every evaluation must be unnecessarily expensive

**Review by awennersteen — CHANGES_REQUESTED** [review]
Further review done offline

File: pyqtorch/converters/store_ops.py (hunk 1 above)
Comment (madagra) [reply pyqtorch/converters/store_ops.py]: @awennersteen I changed the logic and avoid adding the decorator. Now the operation storage is performed only when requested, i.e. in the converter routine. Let me know if it is now acceptable for merge.

**Comment by madagra** [comment]
@awennersteen It should be better now. The actual storage of the operations in the forward pass is only performed when requested in the converter function, otherwise, nothing is done. This also implies that one needs to be a bit careful when adding new operations; now, the routine for storing operations must be explicitly called when needed in the operation implementation.

**Comment by awennersteen** [comment]
LGTM

**Review by awennersteen — APPROVED** [review]

**Pull request merged**